### ISSUE TITLE
Please consider archiving this repository if it's not maintained

### ISSUE THREAD

**Issue by ermik** [tomchentw/react-google-maps#870]
Dear `react-google-maps` authors, I have noticed there is an always growing number of open issues in this repo. And while this is a blight of any popular OSS community, what has raised my present concern was the multitude of PRs that have not been merged or otherwise cleared.

*Please consider archiving this repository or providing a clear notice of maintenance status.* While we make our bets with dependencies, it would be great to know what we are betting on. And no hard feelings, I hope — because we all love the tool you made. ❤️

**Comment by stenrdj** [comment]
@ermik you're absolutely right , please @tomchentw would you keep in view the status of maintainability of this repo !

**Comment by bryan** [comment]
Looks like it's not being maintained. This repo looks more alive: https://github.com/google-map-react/google-map-react
